test/interactions/keyInteractionTests.ts

```diff
@@ -1,302 +1,327 @@
 ///<reference path="../testReference.ts" />
 
 describe("Interactions", () => {
-  describe("KeyInteraction", () => {
-    let svg: d3.Selection<void>;
-    let component: Plottable.Component;
-    let keyInteraction: Plottable.Interactions.Key;
-    let aCode: number;
-    let bCode: number;
-
-    let aCallbackCalled: boolean;
-    let bCallbackCalled: boolean;
-    let aCallback: Plottable.KeyCallback;
-    let bCallback: Plottable.KeyCallback;
-
-    beforeEach(() => {
-      aCode = 65;
-      bCode = 66;
-      component = new Plottable.Component();
-      svg = TestMethods.generateSVG(400, 400);
-      keyInteraction = new Plottable.Interactions.Key();
-      aCallback = () => aCallbackCalled = true;
-      bCallback = () => bCallbackCalled = true;
-      component.renderTo(svg);
-      aCallbackCalled = false;
-      bCallbackCalled = false;
-    });
-
+  describe("Key Interaction", () => {
     describe("onKeyPress", () => {
+      let svg: d3.Selection<void>;
+      let component: Plottable.Component;
+      let keyInteraction: Plottable.Interactions.Key;
+
+      let aKeyCode: number;
+      let aKeyCodeCallbackCalled: boolean;
+      let aKeyCodeCallback: Plottable.KeyCallback;
+
+      beforeEach(() => {
+        component = new Plottable.Component();
+        keyInteraction = new Plottable.Interactions.Key();
+        aKeyCode = 65;
+        aKeyCodeCallbackCalled = false;
+        aKeyCodeCallback = () => aKeyCodeCallbackCalled = true;
+        svg = TestMethods.generateSVG();
+        component.renderTo(svg);
+      });
+
       it("only fires callback for \"a\" when \"a\" key is pressed", () => {
```

Comment:
This does not sounds like the type of test that should be at the front of this test suite.

Instead, can we have something like
it("fires callback for the a key code when that key is pressed)
as the first test and then have
it("does not fire callbacks for key codes other than the pressed key")
as an edge case second test